### ISSUE TITLE
Accesibility colors

### ISSUE THREAD

**Issue by EdwardIrby** [pluralsight/design-system#936]
Text link in `psColorsPrimaryActionTextDarkTheme` on  `psColorsBackgroundDark3` fails the color contrast check

**Comment by sweeting** [comment]
Resolved. The [link color variable used](https://pluralsight.slack.com/archives/C70HPSJPP/p1585007472130600) was the `primaryActionBackground` color instead of the `primaryActionTextDarkTheme` color. 

The correct values are within contrast guidelines. 
![image](https://user-images.githubusercontent.com/48542/77561839-dbc3c480-6e84-11ea-9d09-e2c6ed126c92.png)
